aries-site/src/examples/templates/forms/SignUpExample.js

```diff
@@ -13,6 +13,7 @@ import {
   TextInput,
 } from 'grommet';
 import { FormCheckmark, FormClose } from 'grommet-icons';
+import { useDarkMode } from '../../../utils';
```

Comment:
Changes to this file seem unexpected and unrelated to DateInput

Reply:
strange Ill fix!